v03_pipeline/lib/misc/io.py

```diff
@@ -176,4 +179,4 @@ def write(
     t.write(checkpoint_path)
     t = read_fn(checkpoint_path)
     t = t.naive_coalesce(compute_hail_n_partitions(file_size_bytes(checkpoint_path)))
-    return t.write(destination_path, overwrite=True, stage_locally=True)
```

Comment:
Ran into issues with this flag on the mito re-run w/ a small cluster and running out of disk space...